### ISSUE TITLE
Classic Editor styles constraining editor

### ISSUE THREAD

**Issue by garrett-eclipse** [WordPress/twentytwenty#728]
Hello,

The classic editor has a constrained width of 580px which on large screens has your content in the middle with excessive empty space around it. It would be nice to drop that width on the editor it doesn't seem to provide a benefit.
![Screen Shot 2019-10-02 at 4 03 36 PM](https://user-images.githubusercontent.com/8726005/66088132-dc5f4c00-e52e-11e9-90a5-1425134c7d53.png)

As well the background feels odd here and took a moment to understand it was the editor, I haven't seen previous themes force a background on that editor before and feel it should stay white to conform with other admin inputs.

Thanks

**Comment by joyously** [comment]
It is normal for a theme to change the background color to match the front end, as it does in the block editor.

As for the width constraint, what is the max-width of the text on the front end?

**Comment by garrett-eclipse** [comment]
Thanks Joy, greatly appreciated.

While I understand the editor background and constrained width are to emulate the front-end I still have some concerns, mostly with the width.

1. The constrained width causes variable padding in the editor which can be jarring to the user.
When a user interacts with an input their focus moves to the beginning of it. As you enter text we find it shifted by a variable amount from the bounds of the input requiring the user to change focus unexpectedly. With the block editor the entire editor is constrained so when entering content it appears where you expect it, at the beginning of the input.
*Using custom fields with several text fields prior to an editor gives a good visual of the disruption of the visual input.
<img width="617" alt="Screen Shot 2019-10-02 at 10 56 06 PM" src="https://user-images.githubusercontent.com/8726005/66102655-e8b3cb00-e567-11e9-9315-8ec7f40538a7.png">

2. The styles affect custom fields that utilize wp.editor.
<img width="1344" alt="Screen Shot 2019-10-02 at 10 38 58 PM" src="https://user-images.githubusercontent.com/8726005/66101937-793cdc00-e565-11e9-875c-eaa5c6d589a4.png">
In many cases these aren't rendered on the front-end with the same width constraints and background. 

Myself when creating my first post had to check the Text tab to ensure I hadn't inadvertently centered or introduced tabs into my editor.

While I understand plugins and child themes can suppress or override this behaviour, it feels more disruptive to the user experience than beneficial.

Let me know your thoughts and I'd be happy to create a PR.

**Comment by joyously** [comment]
1. The block editor styles do the exact same thing as Classic editor styles: make it look like front end. Why is this a problem? It is the goal.
Block editor:
![Add New Post Local Current wide-block-editor](https://user-images.githubusercontent.com/7957680/66133003-943a3b00-e5bb-11e9-9016-76a216388cfb.jpg)

Classic editor:
![Add New Post Local Current wide](https://user-images.githubusercontent.com/7957680/66132472-bbdcd380-e5ba-11e9-80b0-686754ca3a1f.jpg)

All you have to do is make your window smaller, if you don't want to see all that empty space:
Classic editor on 1040px window
![Add New Post Local Current 1040](https://user-images.githubusercontent.com/7957680/66132471-bb443d00-e5ba-11e9-96c3-d6d758c14b43.jpg)


2. It is to be expected that every invocation of the TinyMCE editor is styled, but this includes the Classic block, so care should be taken to not style those. In my theme, I used `html:not(.wp-toolbar)` to target the editor preview, but not the admin html element, because the Classic block uses a div instead of an iframe.

**Comment by joyously** [comment]
I forgot to mention that the one thing that seems wrong to me in the wide Classic editor screenshot above, is that the aligned image is not outside of the text like it would be on the front end.

**Comment by garrett-eclipse** [comment]
Thanks Joy,

I appreciate you flagging the block editor, as I mentioned the implementation there constrains the width of the editor instance so when you start typing the text appears where you expect at the beginning of the input. All of the tools (editor buttons) stay right there consistently where they're expected. When using the classic editor the text is constrained and variably displaced from the beginning of the input where the user expects the result of their typing.
So maybe the width can be moved to the editor itself so the user expects their content to be the set width, it starts rendering at the beginning of the input and doesn't give the illusion you've accidentally tabbed or centered your content.

As with the Classic block I don't feel the Custom Fields should be affected as they almost never are output under the same constraints as the post_content, so am thinking at least can be more specific;
`body#tinymce.wp-editor.content`
*I've tested ACF & Pods and the `.content` class on the body is unique to WP, ACF uses `.acf_content` and Pods uses `.pods-form-ui-pods-meta-<name>`

**Comment by joyously** [comment]
That selector might be too specific. It would need some testing with the custom colors generated.

**Comment by andersnoren** [comment]
@garrett-eclipse I agree with @joyously that the classic editor should reflect the front-end of the theme as closely as possible, so I think constraining the width of the text area to match the front-end width is motivated. 

I also agree with you that the classic editor styles shouldn't hit WYSIWYG areas generated by plugins, when possible, so I created a PR for changing the targeting of the classic editor from `body#tinymce.wp-editor` to `body#tinymce.wp-editor.content`.

**Comment by garrett-eclipse** [comment]
Thanks @andersnoren & @joyously I appreciate avoiding custom fields 100%.

And while I still feel constraining width changes the expected user experience, I'll defer to you here as I can always override.

Appreciated
